main.star

```diff
@@ -9,23 +11,20 @@ ARTIFACT_SERVER_IMAGE = "nginx:1.25-alpine"
 RPC_PORT_NUM = 8545
 WS_PORT_NUM = 8546
 
-
+# this is adapted from the optimism monorepo
+# https://github.com/ethereum-optimism/optimism/blob/develop/bedrock-devnet/devnet/__init__.py
 def run(plan):
     uploaded_files = upload_config_and_genesis_files(plan)
 
-    # Question - can this just be the ethereum-package?
+    # his could be just stock geth 
+    # https://github.com/ethereum-optimism/optimism/blob/develop/ops-bedrock/Dockerfile.l1
```

Comment:
this is exactly the image that is being used! If you want a more complicated L1 -we could use the `ethereum-package` that would give you an interop testnet with explorers / fork monitors etc